.github/workflows/build-test.yaml

```diff
@@ -32,9 +32,14 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.10'
-          cache: 'pip'
-      - run: pip install -r requirements.txt
-      - run: pip install -r requirements-dev.txt
+        
+      - uses: actions/cache@v3
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('setup.py') }}-${{ hashFiles('requirements.txt') }}-${{ hashFiles('requirements-dev.txt') }}
```

Comment:
If I understand correctly, this generates a directory for each different version of setup.py, requirements.txt and requirements-dev.txt?


What happens with outdated environments, are they deleted after a while?

Reply:
![image](https://user-images.githubusercontent.com/30818944/227414237-b0eb7782-c871-409e-a6eb-52302c6f0088.png)

Reply:
Nice!